app/views/shared/_nav.html.haml

```diff
@@ -37,7 +37,7 @@
           - else
             = link_to "Sign in with GitHub", user_github_omniauth_authorize_path
 
-      = form_tag search_path, method: "get", role: "search", class: "navbar-form navbar-right" do
+      = form_tag search_path, method: "get", role: "search", class: "navbar-form navbar-right", id: "searchf" do
```

Comment:
What does "searchf" signify? Search form? If so I'd suggest using the full wording so it's clear, i.e. "search-form".